etc/config/scala.amazon.properties

```diff
@@ -0,0 +1,26 @@
+compilers=&scala
+compilerType=scala
+versionFlag=-version
+objdumper=/opt/compiler-explorer/jdk-16.0.1/bin/javap
+instructionSet=java
+defaultCompiler=scalac2136
```

Comment:
Hi! Quick question. Why was this chosen as the default, if a later version exists?

Reply:
Hey
Scala 3.0 was only released less than 2 months ago, so I felt like it was a bit early to make it default.
Also, to make the example work in both versions it needs to be compatible with Scala 2 (so no top-level functions).
With these two bits in mind I thought that latest Scala 2 version would be the best default

Reply:
Makes lots of sense, thank you! I'll leave a comment for future reference so we don't forget why :)